tap_zuora/utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, Optional
+
+
+def make_payload(project: str, query: str, partner_id: str, deleted: Optional[bool] = False) -> Dict:
```

Comment:
Ohh, that's a bummer., but I agree.

If it (and the REST version) could also go on the Client class, that would probably be better, since they're more HTTP oriented things.  If that works, can it be renamed to `make_aqua_payload` and `make_rest_payload`?

Reply:
REST doesn't use any payload only AQuA uses. Do u mean to keep it in utils and rename it to `make_aqua_payload`?

Reply:
reiterating my earlier comment. REST doesn't use any payload while validating the potential base URLs only AQuA uses. So, RESTAPI class has only its own make_payload static method which is not being imported in `client.py`

Only the AQuA's make_payload is being in different places.

Reply:
renamed it to `make_aqua_payload`